DOCS-1889: Mentions that PayPal is non-collecting and consequences (#1001)
MultiSafepay/docs: content/api/payment_methods/post-order-paypal.md

Creates a PayPal [Redirect](/faq/api/difference-between-direct-and-redirect) order.

> {{< alert-notice >}} In PayPal, after a successful payment of a transaction, the order status will be set to _Completed_ and the financial status will remain set to _Initialized_. If the financial status remains _Initialized_, an order cannot be delivered since shipment of an order depends on the financial status. You must ensure that orders are set to _Completed_ for both the order and financial status after a successful payment. {{< /alert-notice >}}

Reviewer: ```suggestion
{{< alert-notice >}} In PayPal, after a successful payment of a transaction, the order status is set to _Completed_ and the financial status remains set to _Initialized_. If the financial status remains _Initialized_, an order cannot be delivered since the shipment of an order depends on the financial status. You must ensure that orders are set to _Completed_ for both the order and financial status after a successful payment. {{< /alert-notice >}}
```
